src/tuf.rs

```diff
@@ -171,13 +183,16 @@ impl<D: DataInterchange> Tuf<D> {
         signed_timestamp: SignedMetadata<D, TimestampMetadata>,
```

Comment:
Consider renaming signed_timestamp (plus the other metadata types) to have a `new_` prefix too. This may or may not be signed correctly.